docs/dev/table/streaming/time_attributes.md

```diff
@@ -284,13 +284,19 @@ public class UserActionSource implements StreamTableSource<Row>, DefinedRowtimeA
 	}
 
 	@Override
-	public String getRowtimeAttribute() {
+	public List<RowtimeAttributeDescriptor> getRowtimeAttributeDescriptors() {
 		// Mark the "UserActionTime" attribute as event-time attribute.
-		return "UserActionTime";
+		// here we create one attribute descriptor of "UserActionTime"
+		RowtimeAttributeDescriptor rowtimeAttrDescr = new RowtimeAttributeDescriptor(
+		    "UserActionTime",
```

Comment:
Still using blank.

Reply:
Hi, I think it's correct now, here is the code view form my patch.
![image](https://user-images.githubusercontent.com/13216107/51024910-ae636180-15c5-11e9-8bc2-5a27f6f4d30d.png)